addon/index.js

```diff
@@ -7,6 +7,11 @@ let squelchedLabels;
 export let squelchErrorHandlerFor = null;
 export let unsquelchAllErrorHandlers = null;
 
+const excludeValue = (array, val) => {
```

Comment:
Hmm, where is this used?

Reply:
It was, uhh… that kind of morning. I'll remove it.